contrib/endpoints/src/api_manager/auth/lib/auth_jwt_validator.cc

```diff
@@ -699,12 +699,19 @@ grpc_jwt_verifier_status JwtValidatorImpl::FillUserInfoAndSetExp(
 
   // Optional field.
   const grpc_json *grpc_json = grpc_jwt_claims_json(claims_);
+
+  char *json_str =
+      grpc_json_dump_to_string(const_cast<::grpc_json *>(grpc_json), 0);
+  user_info->claims = json_str == nullptr ? "" : json_str;
+  if (json_str != nullptr) {
+    gpr_free(json_str);
```

Comment:
move the line above in here? member std::string is default to "", so you just need to assign once.